.github/workflows/build.yml

```diff
@@ -85,6 +115,14 @@ jobs:
           key: sentry-native-Linux-${{ hashFiles('scripts/build-sentry-native.ps1') }}-${{ hashFiles('.git/modules/modules/sentry-native/HEAD') }}
           fail-on-cache-miss: true
 
+      - name: Download sentry-native (Linux musl)
+        if: ${{ (env.CI_PUBLISHING_BUILD == 'true') || (startsWith(matrix.runner.container, 'alpine')) }}
+        uses: actions/cache/restore@v4
+        with:
+          path: src/Sentry/Platforms/Native/sentry-native
+          key: sentry-native-alpine:3.21-${{ hashFiles('scripts/build-sentry-native.ps1') }}-${{ hashFiles('.git/modules/modules/sentry-native/HEAD') }}
```

Comment:
Maybe use `${{ !matrix.runner.container }}` here instead of hard coding `alpine:3.21`?

Reply:
I resorted to temporarily hard-coding it because the macOS publishing job doesn't have `matrix.runner.container` defined. How about defining a helper "target" variable in the matrix so that one could use something prettier like 'Linux-musl' as a cache key? At the same time, it could help selecting a different `.slnf` because the Android SDK Build-Tools contain musl-incompatible prebuilt binaries... :slightly_frowning_face: